xonsh/parsers/base.py

```diff
@@ -1798,7 +1798,10 @@ def p_trailer_quest(self, p):
     def p_subscriptlist(self, p):
         """subscriptlist : subscript comma_subscript_list_opt comma_opt"""
         p1, p2 = p[1], p[2]
-        if p2 is not None:
+        if isinstance(p1, ast.Slice):
+            if p2 is not None:
+                p1 = ast.ExtSlice(dims=[p1, *p2])
```

Comment:
I think ExtSlice should only be used if p2 is not None and any of the elements of p2 or p1 is a Slice, otherwise it should just be a Tuple.